src/pages/ApplyWrite/index.js

```diff
@@ -1,7 +1,221 @@
-import React from 'react';
+import { useFormik } from 'formik';
+
+import styled, { css } from 'styled-components';
 
 const ApplyWritePage = () => {
-  return <div>ApplyWritePage</div>;
+  const TEMP_QUESTIONS = [
+    {
+      id: 1,
+      question:
+        '저희 멋쟁이사자처럼 11기에 지원해 주심에 감사드리며, 지원자분의 지원 동기를 올해의 목표와 연관 지어 서술해 주세요',
+      maxLength: 1000,
+    },
+    {
+      id: 2,
+      question:
+        '저희 멋쟁이사자처럼 11기에 지원해 주심에 감사드리며, 지원자분의 지원 동기를 올해의 목표와 연관 지어 서술해 주세요',
+      maxLength: 700,
+    },
+  ];
+  //추후 서버 통신시에 마운트되면 문항 질문 받아서 렌더링 (props x)
+
+  const initialValues = {
+    firstAnswer: '',
+    secondAnswer: '',
+    thirdAnswer: '',
+    fourthAnswer: '',
+    fifthAnswer: '',
+    sixthAnswer: '',
+    file: '',
+  };
+
+  const order = Object.keys(initialValues);
+
+  const formik = useFormik({
+    initialValues,
+    onSubmit: values => {
+      alert(JSON.stringify(values, null, 2));
+    },
+  });
+
+  const onChange = (e, key, maxLength) => {
+    if (e.target.value > maxLength) {
+      return;
+    }
+    formik.setFieldValue(key, e.target.value);
+  };
+
+  const fileData = formik.values.file;
+
+  return (
+    <>
+      <FileUploadContainer>
+        <BaseTitle>
+          Html 형식으로 된 자기소개서 페이지를 첨부해 주세요. <Star>*</Star>
+        </BaseTitle>
+
+        <FileUploadBorder file={fileData}>
+          <FileUploadLabel htmlFor="file">
+            <FileUploadTitle file={fileData}>{fileData ? `${fileData.name}` : '파일 불러오기'}</FileUploadTitle>
+          </FileUploadLabel>
+        </FileUploadBorder>
+      </FileUploadContainer>
+      <FileUpload
+        id="file"
+        name="file"
+        type="file"
+        accept=".html"
+        onChange={e => {
+          formik.setFieldValue('file', e.currentTarget.files[0]);
+        }}
+      />
+
+      <BreakLine />
+      <form onSubmit={formik.handleSubmit}>
+        {TEMP_QUESTIONS.map((item, index) => {
+          return (
+            <WriteContainer key={index}>
+              <WriteTitle>
+                {` ${index + 1}. ${item.question}`}
+                {/* 추후 서버통신하면 변경요망 */}
+              </WriteTitle>
+              <WriteBox>
+                <WriteArea
+                  id={order[index]}
+                  placeholder="내용을 입력해주세요"
+                  maxLength={item.maxLength}
+                  name={order[index]}
+                  onChange={e => onChange(e, order[index], item.maxLength)}
+                  value={formik.values[order[index]]}
+                />
+                <WriteLength>
+                  {formik.values[order[index]].length}/{item.maxLength}
```

Comment:
요걸 함수 호출로 바꿔보면 어떨까요?
```suggestion
                  {getCharacterCount(item, index)}
```
이런 식으로요!